example_configs/kanboard.md

```diff
@@ -0,0 +1,33 @@
+# configuration for Kanboard
+
+add these to the kanboard `config.php`
+
+Make sure you adapt `LDAP_SERVER` to the correct LDAP endpoint.
+
+You also might have to change `dc=example,dc=com` to whatever your LLDAP is handling, and maybe change the `lldap_kanboard` group name used in `LDAP_USER_FILTER` to identify users of a valid group of yours.
```

Comment:
Can you change `lldap_kanboard` to `kanboard_users` ? The groups starting with `lldap_` are reserved by the implementation for future use.